src/components/tekstomrade/tekstomrade.less

```diff
@@ -0,0 +1,2 @@
+.tekstomrade {
```

Comment:
Denne gjorde kanksje ikke så mye nytte for seg?

Reply:
Nope, fikk aldri behov for den gitt. Sletta hele fila jeg